docs/spectral_extraction.rst

```diff
@@ -40,22 +40,35 @@ mask from scratch and apply it to the data.::
     >>> maskedsubcube = subcube.with_mask(mask)  # doctest: +SKIP
     >>> spectrum = maskedsubcube.mean(axis=(1,2))  # doctest: +SKIP
 
-Aperture extraction using regions
----------------------------------
+Aperture and spectral extraction using regions
+----------------------------------------------
 
-Spectral-cube supports ds9 regions, so you can use the ds9 region to create a
-mask.  The ds9 region support relies on `pyregion
-<https://pyregion.readthedocs.io/en/latest/>`_, which supports most shapes in
-ds9, so you are not limited to circular apertures.
+Spectral-cube supports ds9 and crtf regions, so you can use them to create a
+mask.  The ds9/crtf region support relies on `~regions` , which supports most shapes in
+ds9 and crtf, so you are not limited to circular apertures.
 
-In this example, we'll create a region "from scratch", but you can also use a
-predefined region file using `pyregion.open
-<http://pyregion.readthedocs.io/en/latest/api/pyregion.open.html>`_.::
+In this example, we'll extract a subcube from ds9 region string using
+`~spectral_cube.spectral_cube.SpectralCube.subcube_from_ds9region`::
 
-    >>> shapelist = pyregion.parse("fk5; circle(19:23:43.907,+14:30:34.66, 3\")")  # doctest: +SKIP
-    >>> subcube = cube.subcube_from_ds9region(shapelist)  # doctest: +SKIP
-    >>> spectrum = subcube.mean(axis=(1,2))  # doctest: +SKIP
+    >>> ds9_str = 'fk5; circle(19:23:43.907, +14:30:34.66, 3")'  # doctest: +SKIP
+    >>> subcube = cube.subcube_from_ds9region(ds9_str)  # doctest: +SKIP
+    >>> spectrum = subcube.mean(axis=(1, 2))  # doctest: +SKIP
+
+Similarly, we'll extract a subcube from crtf region string using
+`~spectral_cube.spectral_cube.SpectralCube.subcube_from_crtfregion`::
+
+    >>> crtf_str = 'circle[[19:23:43.907, +14:30:34.66], 3"], coord=fk5, range=[150km/s, 300km/s]'  # doctest: +SKIP
+    >>> subcube = cube.subcube_from_crtfregion(crtf_str)  # doctest: +SKIP
+    >>> spectrum = subcube.mean(axis=(1, 2))  # doctest: +SKIP
+
+We can also use a list of `~regions.Region` object to extract a subcube using
+`~spectral_cube.spectral_cube.SpectralCube.subcube_from_regions`.
+You can convert the ds9 string into `~regions.Region` object using
+`~regions.DS9Parser` or directly read from a file using `~regions.read_ds9`::
```

Comment:
This example doesn't use a string, right?  So maybe just leave it out.

Also, the example as you have it now only uses a single region.  We want to show the use case involving a list of regions, e.g., a circle and a square with partial overlap.